src/ralph/accounts/management/commands/ldap_sync.py

```diff
@@ -0,0 +1,161 @@
+                ldap.SCOPE_SUBTREE,
+                query,
+                serverctrls=[lc],
+            )
+            page_num = 0
+            while True:
+                page_num += 1
+                r_type, r_data, r_msgid, serverctrls = self.conn.result3(msgid)
+                print("Pack of %s users loaded (page %s)" % (
+                    LDAP_RESULTS_PAGE_SIZE,
+                    page_num,
+                ))
+                for item in r_data:
+                    yield item
+                if serverctrls:
+                    if serverctrls[0].cookie:
+                        lc.size = LDAP_RESULTS_PAGE_SIZE
+                        lc.cookie = serverctrls[0].cookie
+                        msgid = self.conn.search_ext(
+                            settings.AUTH_LDAP_USER_SEARCH_BASE,
+                            ldap.SCOPE_SUBTREE,
+                            query,
+                            serverctrls=[lc],
+                        )
+                    else:
+                        break
+                else:
+                    logger.error(
+                        'LDAP::_run_ldap_query\tQuery: %s\t'
+                        'Server ignores RFC 2696 control',
+                    )
+                    sys.exit(0)
+        finally:
+            self._disconnect()
+
+    def _get_users(self):
+        objcls = settings.LDAP_SERVER_OBJECT_USER_CLASS
+        try:
+            user_filter = settings.AUTH_LDAP_USER_FILTER
+        except AttributeError:
+            query = '(objectClass=%s)' % (objcls,)
+        else:
+            query = '(&(objectClass=%s)%s)' % (objcls, user_filter,)
+        ldap_users = self._run_ldap_query(query)
+        return ldap_users
+
+    def _load_backend(self):
+        path = settings.AUTHENTICATION_BACKENDS[0].split('.')
+        module_path = path[:-1]
+        class_name = path[-1]
+        module = __import__('.'.join(module_path), globals(), locals(),
+                            class_name)
+        self.backend = getattr(module, class_name)()
+
+    def check_settings_existence(self):
+        """Check if all needed settings are defined in settings.py"""
+        options = [
+            'AUTH_LDAP_SERVER_URI',
+            'AUTH_LDAP_USER_SEARCH_BASE',
+            'AUTH_LDAP_USER_USERNAME_ATTR',
+            'AUTH_LDAP_PROTOCOL_VERSION',
+            'AUTH_LDAP_BIND_DN',
+            'AUTH_LDAP_BIND_PASSWORD',
+        ]
+        for option in options:
+            if not hasattr(settings, option):
+                logger.error(
+                    'LDAP::check_settings_existence\tSetting %s '
+                    'is not provided.' % option
+                )
+                print("Setting '%s' is not provided." % option)
+                sys.exit(0)
+
+    def handle(self, *args, **kwargs):
+        """Load users from ldap command."""
+        self.check_settings_existence()
+        self._load_backend()
+        print("Syncing...")
```

Comment:
change every print to `self.stdout.write` instead of print in whole file (https://docs.djangoproject.com/en/1.8/howto/custom-management-commands/)